pkg/antctl/antctl.go

```diff
@@ -726,6 +726,33 @@ $ antctl get podmulticaststats pod -n namespace`,
 			commandGroup:        get,
 			transformedResponse: reflect.TypeOf(agentapis.BGPRouteResponse{}),
 		},
+		{
+			use:   "fqdncache",
+			short: "Print fqdn cache",
+			long:  "Print effective fqdn cache information including fqdn name, IP addresses, and expiration time",
+			example: `	Get the list of all fqdn rules currently applied
+			$ antctl get fqdncache
+			Get the list of all fqdn rules currently applied
+			$ antctl get fqdncache --domain DomainName
+			Get the list of all fqdn rules currently applied for a given domain name
```

Comment:
something is off: you are repeating `Get the list of all fqdn rules currently applied` twice. The description should go before the example command.
also maybe add real examples, e.g. with domains `example.com` and `*.antrea.io` (to show that wildcards are supported)